src/Utility/NitroBoost.js

```diff
@@ -0,0 +1,40 @@
+const BotModule = require('../BotModule');
+
+const Message = {
```

Comment:
Variables shouldn't start with an uppercase letter.

Reply:
But we've been doing this all the time for things that we don't treat like variables (like BotModule)

Reply:
It's breaking the naming convention that you can see in commands such as support, whowas, playercount, mcstatus, fry, etc.